sessionctx/variable/sysvar.go

```diff
@@ -620,6 +620,7 @@ var defaultSysVars = []*SysVar{
 	{ScopeSession, TiDBDMLBatchSize, strconv.Itoa(DefDMLBatchSize)},
 	{ScopeSession, TiDBCurrentTS, strconv.Itoa(DefCurretTS)},
 	{ScopeSession, TiDBMaxChunkSize, strconv.Itoa(DefMaxChunkSize)},
+	{ScopeSession, TiDBLogQuery, strconv.Itoa(DefLogQuery)},
```

Comment:
This is a session scope variable. How could the users enable this feature? Most of them do not know it.
I think we could make it a global scope variable or a configuration of the server.

Reply:
This should only be used temporarily and for a single tidb-server.
Make it a configuration require a restart which is inconvenient.

Reply:
I think we should add a comment here. For this variable is a 'server scope' variable instead of a 'session scope' variable.